f1pystats/f1pystats.py

```diff
@@ -27,228 +27,257 @@
+        r_sched = RaceSchedule(schedule_json)
+        race_round = r_sched.get_race_round()
+        race_name = r_sched.get_race_names()
+        race_circuits = r_sched.get_race_circuits()
+        race_schedule_date = r_sched.get_race_schedule_dates()
+        race_schedule_time = r_sched.get_race_schedule_times()
+        race_locality = r_sched.get_race_locality()
+        race_country = r_sched.get_race_countries()
+    
+        return pd.DataFrame(
+            zip(
+                race_round,
+                race_name,
+                race_circuits,
+                race_schedule_date,
+                race_schedule_time,
+                race_locality,
+                race_country,
+            ),
+            columns=[
+                "Round",
+                "Race Name",
+                "Circuit",
+                "Date",
+                "Time",
+                "Locality",
+                "Country",
+            ],
+        )
 
 
 def lap_times(year: int, race_round: int, lap_number: int):
     """Returns the lap times for a specified year, race round and lap number"""
-    json_data = _get_json_content_from_url(
-        f"https://ergast.com/api/f1/{year}/{race_round}/laps/{lap_number}.json"
-    )
-    schedule_json = json_data["MRData"]["RaceTable"]["Races"][0]["Laps"][0]["Timings"]
-    l_times = LapTimes(schedule_json)
-    driver_names = l_times.get_driver_names()
-    driver_positions = l_times.get_driver_positions()
-    driver_timings = l_times.get_lap_time()
-
-    return pd.DataFrame(
-        zip(
-            driver_positions,
-            driver_names,
-            driver_timings,
-        ),
-        columns=[
-            "POS",
-            "Driver",
-            "Lap Time",
-        ],
-    )
+    if year < 1950 and year > 2022:
+        raise ValueError("Only years between 1950 and 2022 are considered as valid value for year")
+    else:
+        json_data = _get_json_content_from_url(
+            f"https://ergast.com/api/f1/{year}/{race_round}/laps/{lap_number}.json"
+        )
+        schedule_json = json_data["MRData"]["RaceTable"]["Races"][0]["Laps"][0]["Timings"]
+        l_times = LapTimes(schedule_json)
+        driver_names = l_times.get_driver_names()
+        driver_positions = l_times.get_driver_positions()
+        driver_timings = l_times.get_lap_time()
+    
+        return pd.DataFrame(
+            zip(
+                driver_positions,
+                driver_names,
+                driver_timings,
+            ),
+            columns=[
+                "POS",
+                "Driver",
+                "Lap Time",
+            ],
+        )
 
 
 def pit_stops(year: int, race_round: int, stop_number: int = 0):
     """Returns the pit stops for a specific race in a season"""
-    if int == 0:
-        json_data = _get_json_content_from_url(
-            f"https://ergast.com/api/f1/{year}/{race_round}/pitstops.json"
-        )
+    if year < 2012 and year > 2022:
```

Comment:
Should be ```or``` instead of ```and```.

Please correct all other occurrences

Reply:
Sure, working on it.